test/expected/size_utils.out

```diff
@@ -50,45 +50,45 @@ SELECT * FROM hypertable_relation_size_pretty('"public"."two_Partitions"');
-        4 | "_timescaledb_internal"."_hyper_1_4_chunk" | {timeCustom,device_id} | {bigint,text}             | {NULL,_timescaledb_internal.get_partition_hash} | {"['1257892416000000000','1257895008000000000')","[,1073741823)"} | 8192 bytes | 96 kB      | 8192 bytes | 112 kB
+ chunk_id |              chunk_table               |  partitioning_columns  | partitioning_column_types |           partitioning_hash_functions           |                              ranges                               | table_size | index_size | toast_size | total_size 
+----------+----------------------------------------+------------------------+---------------------------+-------------------------------------------------+-------------------------------------------------------------------+------------+------------+------------+------------
+        1 | _timescaledb_internal._hyper_1_1_chunk | {timeCustom,device_id} | {bigint,text}             | {NULL,_timescaledb_internal.get_partition_hash} | {"['1257892416000000000','1257895008000000000')","[1073741823,)"} | 8192 bytes | 112 kB     | 8192 bytes | 128 kB
+        2 | _timescaledb_internal._hyper_1_2_chunk | {timeCustom,device_id} | {bigint,text}             | {NULL,_timescaledb_internal.get_partition_hash} | {"['1257897600000000000','1257900192000000000')","[1073741823,)"} | 8192 bytes | 104 kB     | 8192 bytes | 120 kB
+        3 | _timescaledb_internal._hyper_1_3_chunk | {timeCustom,device_id} | {bigint,text}             | {NULL,_timescaledb_internal.get_partition_hash} | {"['1257985728000000000','1257988320000000000')","[1073741823,)"} | 8192 bytes | 96 kB      | 8192 bytes | 112 kB
+        4 | _timescaledb_internal._hyper_1_4_chunk | {timeCustom,device_id} | {bigint,text}             | {NULL,_timescaledb_internal.get_partition_hash} | {"['1257892416000000000','1257895008000000000')","[,1073741823)"} | 8192 bytes | 96 kB      | 8192 bytes | 112 kB
 (4 rows)
 
 SELECT * FROM indexes_relation_size('"public"."two_Partitions"');
-                    index_name                    | total_bytes 
---------------------------------------------------+-------------
- public.two_Partitions_device_id_timeCustom_idx   |       65536
- public.two_Partitions_timeCustom_device_id_idx   |       65536
- public.two_Partitions_timeCustom_idx             |       65536
- public.two_Partitions_timeCustom_series_0_idx    |       65536
- public.two_Partitions_timeCustom_series_1_idx    |       65536
- public.two_Partitions_timeCustom_series_2_idx    |       40960
- public.two_Partitions_timeCustom_series_bool_idx |       49152
+                     index_name                     | total_bytes 
+----------------------------------------------------+-------------
+ public."two_Partitions_device_id_timeCustom_idx"   |       65536
```

Comment:
why are index names quoted here? Seems inconsistent with other functions that return unquoted names. Also, why is schema unquoted in that case? I guess PG automatically quotes if the name starts capitalized, but this is not the case here.

Reply:
It quotes if any part of the name is capitalized since identifiers are by default all lowercase. There are 2 uppercase letters so it needs quoting.